packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -85,6 +85,40 @@ void main() {
     );
   }, skip: isBrowser); // https://github.com/flutter/flutter/issues/44572
 
+  testWidgets('Custom Padding', (WidgetTester tester) async {
```

Comment:
We should also be testing the default for M3 and M2 and verifying that the default can be overridden with the theme.